torch/_decomp/decompositions.py

```diff
@@ -4321,7 +4321,14 @@ def scaled_dot_product_flash_attention_for_cpu(
     )
 
     output, attn = aten._scaled_dot_product_attention_math.default(
-        query, key, value, None, dropout_p, is_causal, None, scale=scale
+        query,
+        key,
+        value,
+        attn_mask=attn_mask,
```

Comment:
Can you update the test in https://github.com/pytorch/pytorch/blob/main/test/test_decomp.py#L921